chapter_11/ex01_libfuzzer_static_lib/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+# CMakeLists file for the Chapter 11 Example nr. 01:
+# Static library fuzzing with libfuzzer
+#
+# SPDX-License-Identifier: MIT
+
+project(
+  ch11_ex01_libfuzzer_static
+  VERSION 1.0
+  DESCRIPTION "Chapter 11 Example 01, fuzzing of a static library with libfuzzer"
+  LANGUAGES CXX)
+
+# Define an executable target named `ch11_ex01_libfuzzer_static` 
+add_library(ch11_ex01_libfuzzer_static STATIC)
+# Specify source files for target named `ch11_ex01_libfuzzer_static`
+target_sources(ch11_ex01_libfuzzer_static PRIVATE src/library.cpp)
+# Specify the include directories for the target named `ch11_ex01_libfuzzer_static`
+target_include_directories(ch11_ex01_libfuzzer_static PUBLIC include)
+# Request compile features for target named `ch11_ex01_libfuzzer_static`.
+# Requested `cxx_std_11` feature determines the minimum C++ standard required
+# to build this target. It's PRIVATE, since it is a build-time requirement only.
+target_compile_features(ch11_ex01_libfuzzer_static PRIVATE cxx_std_11)
+# Include the fuzzing target
+add_subdirectory(fuzz)
```

Comment:
Would it make sense to add any tests that execute the code generated with libfuzzer? I'm not super familiar with it, but as far as I know some of the issues will only be detected during runtime

Reply:
I already added some conditional tests for the sanitizer examples. I think adding the fuzzing examples to the santizers workflow might work. 
https://github.com/PacktPublishing/CMake-Tips-and-Tricks/blob/e533b5e136c484ad205cdb8e85c7a0323496fbe0/chapter_7/combined_sanitizer_example/CMakeLists.txt#L82